cw_evm/lib/evm_chain_wallet.dart

```diff
@@ -575,7 +575,7 @@ abstract class EVMChainWalletBase
         totalAmount = (currencyBalance.balance - estimatedFeesForTransaction);
       }
 
-      // check the fees on the base currency (Eth/Polygon)
+      // check the fees on the base currency
```

Comment:
let's check the fee calculation for all of them as Jack reported that we have a very high fee for Base